assets/script.js

```diff
@@ -0,0 +1,203 @@
+    // AjAX request to API to retrieve horoscope info
+    $.ajax({
+        type: 'POST',
+        url: 'https://aztro.sameerkumar.website?sign=aries&day=today',
+        url: queryURL,
+        success: function (horData) {
+            console.log(horData);
+            updatePage(horData, sign);
+        }
+    });
+};
+// this function will be executed when the page is loaded
+$(document).ready(function () {
+
+    $('.sidenav').sidenav();
+    console.log(window.document.title);
+    if (window.document.title === "Tarot") {
+        console.log("On Tarot Page");
+        tarot();
+    }
+    else if (window.document.title === "Astrozoders") {
+        index = true;
+        var name = JSON.parse(localStorage.getItem('name'));
+        console.log("On Main Page ", name);
+        if (name !== null) {
+            historySign = JSON.parse(localStorage.getItem('sign'));
+            historyName = JSON.parse(localStorage.getItem('name'))
+            $("#name").val(historyName);
+            console.log(historySign, " ", historyName);
+            sign = historySign;
+            buildURL(sign);
+
+        }
+    }
+    else {
+        console.log("On Sign Page");
+        var sign = window.document.title;
+        console.log("Sign from title is: ", sign);
+        console.log("index flag set to: ", index);
+        index = false;
+        console.log("index flag set to: ", index);
+        buildURL(sign);
+
+    }
+})
+
+// Event listner for sogns 
+$(".sign").on("click", function () {
+    localStorage.clear();
+
+    var sign = this.id;
+    buildURL(sign);
+
+});
+
+// Evennt listner for tarot cards
+$(".tarot").on("click", function () {
+    if (tarotID === this.id) {
+
+        console.log("Card Closed");
+        tarotID = "";
+        $(".modal-content").children("p").empty();
+        $(this).attr("data-state", "close");
+    }
+    else {
+        tarotID = this.id
+        console.log($(this).attr("data-state"));
+        var index = this.id;
+
+        console.log(this);
+        console.log($(this).children().children("p").html());
+
+        var pEl = ($(this).children().children("p"));
+        console.log("p Element: ", pEl, "length: ", pEl[0].length);
+        $(".modal-content").children("p").empty();
+        $(this).children().children("p").html(tarotName[index])
+        $(".modal-content").append("<p>" + tarotDesc[index] + "</p>");
+        console.log($(".modal-content"));
+        if (($(this).attr("data-state")) === "close") {
+            $('.modal').modal();
+            $('#modal1').modal('open');
+            $(this).attr("data-state", "open");
+        }
+        else {
+            $(this).attr("data-state", "close");
+        }
+    }
+});
+
+// Event listner for clear button 
+$("#clear").on("click", function () {
+    console.log("inside ckear");
+    localStorage.clear();
+    clear();
+    var name = "";
+    $("#name").val(name);
```

Comment:
you could have just done


```
$('#name').val('');
```

try to write as few lines as possible unless it's necessary for clarity